frontend/src/components/ApprovalRequests.vue

```diff
@@ -1,57 +1,152 @@
+                v-toolbar(flat='', color='white')
+                    v-toolbar-title
+                        v-tabs(v-model="current_course")
+                            v-tab(v-for ="course in courses" :key = "course.id" ) {{ course.code }}
+                    v-spacer
+                    v-dialog(v-model='dialog', max-width='1000px')
+                        v-card.rounded-card
+                            v-container.pa-2
+                                v-card-title
+                                    v-row
+                                        h4 Submission Details
+                                div
+                                    div
+                                        v-card-text
+                                            h3 Completed TA Duties
+                                            v-row
+                                                v-textarea(
+                                                    name="input-7-1"
+                                                    filled
+                                                    background-color="white"
+                                                    label=""
+                                                    auto-grow
+                                                    rounded
+                                                    :value="DialogBoxItem.duties_completed"
+                                                    outlined
+                                                    readonly )
+                                            h3 Add Comments
+                                            v-row
+                                                v-textarea(
+                                                    name="input-7-1"
+                                                    filled
+                                                    background-color="white"
+                                                    label=""
+                                                    auto-grow
+                                                    rounded
+                                                    v-model="DialogBoxItem.comment"
+                                                    outlined
+                                                    )
+                                    div(align="right")
+                                        v-btn.ma-2(width=200 rounded outlined color="blue" right @click="close") Close
+                                        v-btn.ma-2(width=200 rounded color="#00948E" right @click="save") Save
+            template(v-slot:item.action='{ item }')
+                v-icon.mr-2(small='', @click='editItem(item)')  View Form
+        div.pa-2 By clicking Approve,you are certifying that the above student(s) have successfully performed the ta duty equivalent to 8 hours per week.
+        v-btn.ma-2(width=200 rounded color="#00948E" class="white--text" right @click.stop="approveDialog = true") Approve TA Release
```

Comment:
Use theme colors only, don't hard code colors, add this to the theme instead if you really want this color.

Reply:
Also, the class can be defined outside the parenthesis, `v-btn.ma-2.white--text`